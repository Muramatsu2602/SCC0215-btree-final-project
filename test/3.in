2 csv/linha3.csv bin/linha3.bin
