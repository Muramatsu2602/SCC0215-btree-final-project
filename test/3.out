5294.280000
